db/migrations/postgresql/app/V2.0.0.4__table_support_requests.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS app.support_requests (
```

Comment:
Is there a specific reason why the schema/table names are not in quotes here (or are quoted in other places)?

Reply:
That feels as a minor issue as both syntaxes are valid

Reply:
Yes, that's definitely a nitpick, feel free to ignore it @Christian-Vasilev

Reply:
We should keep it consistent. I will update and replace everything in quotes, thanks for noticing it.

Reply:
You can ignore the migration for the contact module. I already fixed it in the other repo https://github.com/podkrepi-bg/module-contact/tree/main/database/migrations